program/tests/gear.rs

```diff
@@ -35,3 +15,9 @@ async fn api_timeout() {
         ))
     ));
 }
+
+#[test]
+fn paths() {
+    assert!(PathBuf::from(env::bin("gear")).exists());
+    assert!(PathBuf::from(env::bin("gear-node")).exists());
```

Comment:
Seems that we are planning to rename gear-node to gear, so better to rename current gear to gprogram, for example